doc/z-chap03.xml

```diff
@@ -165,7 +165,9 @@
   <Section Label = "creating-bipartitions">
     <Heading>Creating bipartitions</Heading>
   There are several ways of creating bipartitions in &GAP;, which are
-  described in this section.
+  described in this section. The maximum degree of a bipartition is set as 
+  2 ^ 31. In reality, bipartitions of degrees as small as 2 ^ 24 are unlikely
+  to work because they require too much memory.
```

Comment:
Also it seems that you limit this to `2 ^ 31 - 1` so this comment is not very accurate.